website/components/GoalTabAdd.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <form class="goal-form" @submit.prevent="handleSubmit">
+    <select id="goal-status" v-model="form.status" class="status">
+      <option
+        selected
+        disabled
+        hidden
+        default
+        value=""
+        class="status__default"
+      >
+        Status
+      </option>
+      <option v-for="option in options.status" :key="option.type" :value="option.type" class="status__option">
+        {{ option.text }}
+      </option>
+    </select>
+    <div class="title">
+      <input v-model="form.title" name="title" placeholder="Add task" class="title__field">
+    </div>
+    <select id="goal-type" v-model="form.type" class="type">
+      <option
+        selected
+        disabled
+        hidden
+        default
+        value=""
+        class="type__default"
+      >
+        Goal Type
+      </option>
+      <option v-for="option in options.type" :key="option.type" :value="option.type" class="type__option">
+        {{ option.text }}
+      </option>
+    </select>
+  </form>
+</template>
```

Comment:
Can be merged with `GoalTabView` but would make logic complected. Will create a dedicated ticket for it, after discussing how to go about it.